### PR TITLE
Bump lodash version

### PR DESCRIPTION
**Pull request by jairo-bc**
## What? Why?

Snyk suggestions 
https://app.snyk.io/org/bigcommerce/project/e947f4e9-85ef-4858-a86a-82bcdd6b2452/pr-check/350f32d1-efcd-42b0-8a4a-bea671eee6be

## How was it tested?

`npm test`
----

cc @bigcommerce/storefront-team

### CODE REVIEW

**Review by bc-max — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 4.5.3
+- Bump lodash version to 4.17.21 ([#x](https://github.com/bigcommerce/paper-handlebars/pull/x))
```
Comment (bc-jz) [inline CHANGELOG.md:6]: Should this have a pull request number instead of `x` in the URL?

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "handlebars-helpers": "0.8.4",
     "handlebars-utils": "^1.0.6",
     "he": "^1.2.0",
-    "lodash": "^4.17.13",
+    "lodash": "^4.17.21",
```
Comment (bc-jz) [inline package.json:33]: what is lodash used for in handlebars?

Comment (jairo-bc) [reply package.json:33]: It's an utility library, mostly used for iterating some different structures, objects.
I believe it's possisble to get rid of it

File: CHANGELOG.md (hunk 1 above)
Comment (jairo-bc) [reply CHANGELOG.md]: nice catch!

File: package.json (hunk 2 above)
Comment (bc-jz) [reply package.json:33]: Is that something you would advise doing if possible? Maybe a follow up if not simple to strip?

Comment (jairo-bc) [reply package.json:33]: I created a follow up ticket: https://jira.bigcommerce.com/browse/STRF-9673

**Review by bc-jz — APPROVED** [review]
LGTM

**Pull request merged**